Makefile

```diff
@@ -14,7 +14,7 @@ BASH_DEV     := $(COMPOSE_DEV) $(BASH) -c
 BASH_TEST    := $(COMPOSE_TEST) $(BASH) --login -c
 SPEC_PATH    := spec/
 DB		     := "bin/rails db:setup db:migrate"
-LINT    	 := "bin/rails lint"
+LINT    	 := "bin/rails lint['$(files)']"
```

Comment:
How does this effect running lint locally?

Reply:
Tested `make lint` locally and it still works 👍 .

Did some further testing:
- `make files= lint`... lints all files👍 
- `make files='' lint` ... lints all files 👍 
- `make files='app/controllers/application_controller.rb' lint`... lints the one file 👍